src/styles/getTooltipStyles.js

```diff
@@ -0,0 +1,198 @@
+            borderRadius: variables.componentBorderRadiusSmall,
+            ...tooltipVerticalPadding,
+            ...spacing.ph2,
+
+            // Because it uses absolute positioning, the top-left corner of the tooltip is aligned
+            // with the top-left corner of the wrapped component by default.
+            // So we need to shift the tooltip vertically and horizontally to position it correctly.
+            //
+            // First, we'll position it vertically.
+            // To shift the tooltip down, we'll give `top` a positive value.
+            // To shift the tooltip up, we'll give `top` a negative value.
+            top: shouldShowBelow
+
+                // We need to shift the tooltip down below the component. So shift the tooltip down (+) by...
+                ? componentHeight + POINTER_HEIGHT + manualShiftVertical
+
+                // We need to shift the tooltip up above the component. So shift the tooltip up (-) by...
+                : -(tooltipHeight + POINTER_HEIGHT) + manualShiftVertical,
+
+            // Next, we'll position it horizontally.
+            // To shift the tooltip right, we'll give `left` a positive value.
+            // To shift the tooltip left, we'll give `left` a negative value.
+            //
+            // So we'll:
+            //   1) Shift the tooltip right (+) to the center of the component,
+            //      so the left edge lines up with the component center.
+            //   2) Shift it left (-) to by half the tooltip's width,
+            //      so the tooltip's center lines up with the center of the wrapped component.
+            //   3) Add the horizontal shift (left or right) computed above to keep it out of the gutters.
+            //   4) Lastly, add the manual horizontal shift passed in as a parameter.
+            left: ((componentWidth / 2) - (tooltipWidth / 2)) + horizontalShift + manualShiftHorizontal,
+        },
+        tooltipTextStyle: {
+            color: themeColors.textReversed,
+            fontFamily: fontFamily.GTA,
+            fontSize: tooltipFontSize,
+        },
+        pointerWrapperStyle: {
+            position: 'absolute',
+
+
+            // By default, the pointer's top-left will align with the top-left of the wrapped component.
+            //
+            // To align it vertically, we'll:
+            //
+            //   Shift the pointer up (-) by its height, so that the bottom of the pointer lines up
+            //   with the top of the wrapped component.
+            //
+            //   OR if it should show below:
+            //
+            //   Shift the pointer down (+) by the component's height,
+            //      so that the top of the pointer aligns with the bottom of the component.
+            //
+            // Always add the manual vertical shift passed in as a parameter.
+            top: shouldShowBelow ? componentHeight + manualShiftVertical : manualShiftVertical - POINTER_HEIGHT,
```

Comment:
Is `manualShiftVertical` a required parameter? If not, should this have some kind of default value?